src/sticker-builder/stickers/hashtag.sticker.ts

```diff
@@ -0,0 +1,33 @@
+import { InstaSticker } from './insta-sticker';
+
+export interface HashtagStickerOptions {
+  tagName: string;
+  width?: number;
+  height?: number;
+}
+
+export class HashtagSticker extends InstaSticker {
+  public options: HashtagStickerOptions;
+
+  public constructor(options: HashtagStickerOptions) {
+    super(options.width || 0.47, options.height || 0.11);
```

Comment:
Having `super` call and 2 repeated lines with `delete` in almost every `InstaSticker`  descendant hints that it is not perfect design decision to store options as class attribbute.
Why not to use the class itself as a storage? In this case all the descendants will not even need a constructor.
I didn't notice it the first time, sorry bout that.

Reply:
Stickers is pretty-like feeds

Reply:
For stickers with more options i kept the options or should it modify it to use the class-transformer? But then I'd have to call Object.entries in the toJSON method.

Reply:
In terms of design there is no difference between 1 option and 99 options :)
If you find yourself using a constructor just to write its arguments to the class attributes without any modification - it's time to use native TS features and class-transformer.
You can replace lodash `defaults` with native TS default values for class attributes.
Then your class automatically turns into input interface for short-hand functions in `StickerBuilder`. And then you just build the class with `class-transformer`.

There's nothing bad in Object.entries

Reply:
Even more - you shouldn't implement `.toJSON` in every descendant. Implement it only once in parent using `Object.entries` or smth

Reply:
https://github.com/typestack/class-transformer#classtoplain